src/MCPClient/lib/settings/common.py

```diff
@@ -74,6 +77,23 @@
 max_overflow = 40
 port = 3306
 engine = django_mysqlpool.backends.mysqlpool
+
+[email]
+backend = django.core.mail.backends.console.EmailBackend
+host = smtp.gmail.com
+host_password =
+host_user = your_email@example.com
+port = 587
+ssl_certfile =
+ssl_keyfile =
+use_ssl = False
+use_tls = True
+file_path =
+amazon_ses_region = us-east-1
+default_from_email = webmaster@example.com
+subject_prefix = [Archivematica]
+timeout = 300
```

Comment:
The original default was `None` but config doesn't support `None` values for integers so set to a reasonably high value instead.